dlhub_sdk/client.py

```diff
@@ -111,14 +113,14 @@ def __init__(self, dlh_authorizer: Optional[GlobusAuthorizer] = None,
                                       no_local_server=kwargs.get("no_local_server", True),
                                       no_browser=kwargs.get("no_browser", True))
         self._search_client = globus_sdk.SearchClient(authorizer=search_authorizer,
-                                                      http_timeout=5 * 60)
+                                                      transport_params={"http_timeout": 5 * 60})
```

Comment:
Why not use the user-provided timeout here? FWIW, I don't think there's anything anything special about the 5-minute timeout here.

Reply:
oh gotcha! Happy to change it then, I wasn't sure :)